src/Assetic/Filter/BaseProcessFilter.php

```diff
@@ -48,10 +48,14 @@ protected function createProcessBuilder(array $arguments = array())
         return $pb;
     }
 
-    protected function mergeEnv(ProcessBuilder $pb)
+    protected function mergeEnv(Process $process)
```

Comment:
The Process component allows inheriting the env on its own now (which was not the case in 2.1, which is why this logic exists)